src/JuliaSyntaxHighlighting.jl

```diff
@@ -1,8 +1,7 @@
 module JuliaSyntaxHighlighting
 
 import Base: JuliaSyntax, AnnotatedString, annotate!
-import Base.JuliaSyntax: var"@K_str", Kind, Tokenize, tokenize
-import .Tokenize: kind, untokenize
+import Base.JuliaSyntax: var"@K_str", Kind, GreenNode, parseall, kind, flags
```

Comment:
You can just use `import Base.JuliaSyntax: @K_str, ...` here - `var` isn't required for macros in imports :)  import/using/export/public are the only places where macro names are names, not macro calls.

Reply:
Ah yes, thanks. I've just got used to doing `const var"@..." = X.var"@..."` in a bunch of code I've been writing.